### PR TITLE
client STATE updates

### LINKED ISSUE #41: Dynamic "look" panel in UI
To increase discoverability, there should be a panel in the UI that behaves like a perpetually run "look" command.

- [ ] the current room name
- [ ] the current room description
- [ ] the current list of objects in the room, updated as needed

### PR DESCRIPTION
**Pull request by modgethanc**
this PR:

* handles STATE data from the server
* updates here panel with current `/look` information, taken from STATE (closes #41, completes last item of #13)
* updates player info panel with STATE (completes second to last item of #13)

a small issue:

the initial STATE push from the server doesn't get received by the client until after the client sends something after login; i'm not sure how the asyncio stuff works here, so i don't know how to address this. ideally, that STATE info would come in and update all the info panels without the user having to send a command.

screenshot_2018-06-08_00-20-49

### CODE REVIEW

**Comment by vilmibm** [comment]
:heart_eyes: :heart_eyes: :heart_eyes: :heart_eyes: :heart_eyes: :heart_eyes: :heart_eyes: :heart_eyes: :heart_eyes: :heart_eyes: :heart_eyes: :heart_eyes: :heart_eyes: :heart_eyes: :heart_eyes: :heart_eyes: will review this after my current fugue state

File: client/tmclient/screens.py [hunk 1]
```diff
@@ -253,16 +253,61 @@ def refresh_tabs(self):
         self.tab_headers = urwid.Columns(headers)
         self.header = self.tab_headers
 
-    def here_info(self):
-        room_name = self.room.get("name")
-        info = "[{}]".format(room_name)
+    def update_state(self, raw_state):
+        #self.game_walker.append(urwid.Text(raw_state))
+        #self.game_walker.set_focus(len(self.game_walker)-1)
```
Comment (vilmibm) [inline client/tmclient/screens.py]: Is there a reason to keep the commented code around?

File: client/tmclient/screens.py [hunk 2]
```diff
@@ -253,16 +253,61 @@ def refresh_tabs(self):
...
+            contents.append("no one but yourself")
+        else:
+            for o in room.get("contains"):
+                contents.append(o.name)
+
+        lines = [
+                urwid.Text("[{}]".format(room.get("name")), align='center'),
+                urwid.Text("{}\n".format(room.get("description"))),
+                urwid.Text("You see here ({pop}): {contents}".format(
+            pop=len(contents), contents=', '.join(contents)))
```
Comment (vilmibm) [inline client/tmclient/screens.py]: could use some indentation here for readability

**Comment by modgethanc** [comment]
* removed commented cruft
* added indentation at your marked spot; i'm not sure if this is exactly what you meant?

**Comment by vilmibm** [comment]
that was exactly it! thanks again!!!

**Pull request merged**